veles/secrets/pgpass/pgpass.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pgpass provides an extractor for identifying secrets in .pgpass files.
+package pgpass
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/google/osv-scalibr/extractor/filesystem"
+	"github.com/google/osv-scalibr/inventory"
+	"github.com/google/osv-scalibr/plugin"
+)
+
+// Pgpass is a Veles Secret that holds relevant information for a [Postgres Pgpass](https://www.postgresql.org/docs/current/libpq-pgpass.html).
+type Pgpass struct {
+	Hostname string
+	Port     string
+	Database string
+	Username string
+	Password string
+}
+
+const (
+	// Name is the unique name of this extractor.
+	Name = "secrets/pgpass"
+)
+
+var (
+	// pgpassRe is a regular expression that matches the content of a pgpass file entry
+	//
+	// Every entry in the pgpass file is composed by the following fields:
```

Comment:
Can you add a link to some documentation about the pgpass format here? Ideally one that explains that passwords can't use a single *